### PR TITLE
Feature/KNDP-42: Applied same theme as Docusaurus.io with empty Intro,empty home page, header and items

### PR DESCRIPTION
**Pull request by Enotmd**
image
image
"screenshots from local testing"

Project website theme and basic pages:
-Applied to website same theme as Docusaurus.io.
-Empty home page with header and items on it like requested
-Created Docs section with temporarily empty Introduction and  Installation on Linux Platform.

### CODE REVIEW

File: website/docs/getting started/install/linux.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-#
+---
+id: linux
+title: KNDP for Linux
+sidebar_posiion: 2
+---
+
 # <div align="center">   Install KNDP  </div>
```
Comment (art10m) [inline website/docs/getting started/install/linux.md:9]: @Enotmd do not use naming of folder with spaces

File: website/sidebars.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  myAutogeneratedSidebar: [
+    // highlight-start
```
Comment (art10m) [inline website/sidebars.js:3]: @Enotmd Remove unnecessary comments

File: website/sidebars.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    {
+      type: 'autogenerated',
+      dirName: '.', // '.' means the current docs folder
```
Comment (art10m) [inline website/sidebars.js:6]: @Enotmd Remove unnecessary comments

File: website/sidebars.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    },
+    // highlight-end
```
Comment (art10m) [inline website/sidebars.js:8]: @Enotmd Remove unnecessary comment

File: website/sidebars.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  myAutogeneratedSidebar: [  
+    {
+      type: 'autogenerated',
+      dirName: '.', // '.' means the current docs folder
```
Comment (art10m) [inline website/sidebars.js:5]: @Enotmd You forgot one comment

**Review by art10m — CHANGES_REQUESTED** [review]

**Review by web7evgheni — APPROVED** [review]

**Pull request merged**